src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,21 @@
 package core.basesyntax;
 
-public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+import java.util.Random;
+
+public enum ColorSupplier {
```

Comment:
Do not mix enum Color and ColorSupplier class
![image](https://user-images.githubusercontent.com/20326858/202423383-6903bb62-0da9-4ba3-a5ab-ebd3591f697d.png)